### PR TITLE
initial code commit

### PR DESCRIPTION
**Pull request by otaviojava**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  ~   The Eclipse Public License is available at http://www.eclipse.org/legal/epl-v10.html
+  ~   and the Apache License v2.0 is available at http://www.opensource.org/licenses/apache2.0.php.
+  ~
+  ~   You may elect to redistribute this code under either of these licenses.
+  ~
+  ~   Contributors:
+  ~
+  ~   Otavio Santana
+  ~
+  -->
```
Comment (Crell) [inline pom.xml]: Pretty much all of our sample code is under MIT.  This should be the same.

Comment (Crell) [reply pom.xml]: Also, we have no person names on copyright statements.

File: spring/spring-boot-maven-mysql/.platform/local/.gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+/
```
Comment (Crell) [inline spring/spring-boot-maven-mysql/.platform/local/.gitignore]: Huh.  The .platform/local directory shouldn't even be commitable.  It should be excluded by `.gitignore`.  I'm not even sure how you got these in there. :smile: Remove please.

File: spring/spring-boot-maven-mysql/README.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Spring Boot Maven template for Platform.sh
+
+This project provides a starter kit for Spring Boot Maven with MySQL projects hosted on [Platform.sh](http://platform.sh).
+
+We include recommended .platform.app.yaml and .platform files that should suffice for most use cases. You are free to tweak them as needed for your particular site.
```
Comment (Crell) [inline spring/spring-boot-maven-mysql/README.md:5]: File names should be backtick quoted.

File: pom.xml (hunk 1 above)
Comment (otaviojava) [reply pom.xml]: my bad, thank you

File: pom.xml [hunk 4]
```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Crell) [inline pom.xml:1]: I'm not clear how this file gets used.  Each of the applications is self-contained and has its own `pom.xml` file in addition to `.platform.app.yaml`,  no?

Comment (otaviojava) [reply pom.xml:1]: This file is to show applications available using `mvn clean`, furthermore, to we use it on the PRs.
Therefore, when I update the library and update the version. It'll compile app applications to make a green build in the PR.

**Comment by Crell** [comment]
I'm still not clear on how having 5 nearly-identical applications, each connecting to one service, is a more useful and maintainable demo than one application that connects to 5 services.  Neither one is especially useful directly on its own.

**Comment by otaviojava** [comment]
@Crell 
Maybe that is a Java culture.
But we love to have applications copy/paste and ready to go:

* https://github.com/apache/tomee/tree/master/examples
* https://github.com/eugenp/tutorials
* https://github.com/manorrock/jakartaee-examples
* https://github.com/eclipse-ee4j/jakartaee-tutorial-examples
* https://github.com/JNOSQL/artemis-demo/tree/master/artemis-demo-java-se

Plus that will be useful to my biweekly posts about Java.

**Pull request merged**